docs/OPERATIONS.md

```diff
@@ -0,0 +1,274 @@
+# Deploy a Strategy / Vault for V2
+
+**Note**: This [repo](https://github.com/iearn-finance/chief-multisig-officer) is encouraged to create multiple scripts for governance and dev multisig execution of complex transactions.
+
+**IMPORTANT**: you should **NOT** create a new release with a test vault, it will be endorsed as a production.
+
+## Process for new vault release
+
+- Check latest version in `v2.registry.ychad.eth` against the planned new release vault to be sure its an updated version.
+- Deploy vault for production using the new version (this should not be an experimental vault since it will be endorsed with this process).
+- Set governance to `ychad.eth`:
+
+  ```python
+  vault.setGovernance(0xfeb4acf3df3cdea7399794d0869ef76a6efaff52)
+  ```
+
+- Let multisig accept governance:
+
+  ```python
+  vault.acceptGovernance()
+  ```
+
+- Let governance create a new release on `v2.registry.ychad.eth`:
+
+  ```python
+  registry.newRelease(vault)
+  ```
+
+**Note**: Last two steps may need to be done in different transactions since it sometimes can fail in practice using multisig from Gnosis.
+
+## Before deployment
+
+- Coordinate with Core Dev strategist for getting a review on [board](https://github.com/orgs/iearn-finance/projects/5).
+- Complete peer review by at least 2 strategists.
+- Check if `want` token has a deployed vault already (>=v0.3.0) and coordinate to use that first if possible.
+- If a new vault is needed, deploy it using the registry:
+
+  ```python
+  registry.newExperimentalVault(want, you, multisig, treasury, "", "")
+  ```
+
+  - args: token, gov, guardian, rewards, name, symbol
+  - `multisig` (strategists) = '0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7'
+  - gov for testing = '0x846e211e8ba920b353fb717631c015cf04061cc9' (dev.ychad.eth)
```

Comment:
And where is this one coming from?

Reply:
That's the data to replace the variable names here:

registry.newExperimentalVault(want, you, multisig, treasury, "", "")

Reply:
Rewrote it in a more script-like style